quantized_mesh_encoder/normals.py

```diff
@@ -44,8 +44,10 @@ def compute_vertex_normals(positions, indices):
     return normalized_vertex_normals
 
 
-def signNotZero(v):
-    return -1.0 if v < 0.0 else 1.0
+def sign_not_zero(arr):
+    """A variation of np.sign that coerces 0 to 1"""
+    signed = np.sign(arr)
+    return np.where(signed == 0, 1, signed)
```

Comment:
This is two steps instead of one, and will be slower than what I suggested. It's easy to check:

```py
import numpy as np
arr = np.random.random((1_000_000)) * 2 - 1

def a(arr):
    signed = np.sign(arr)
    return np.where(signed == 0, 1, signed)

def b(arr):
    return np.where(arr < 0, -1, 1)

%timeit a(arr)
# 7.7 ms ± 68.8 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)

%timeit b(arr)
# 5.75 ms ± 66.2 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)
```

```suggestion
    return np.where(arr < 0.0, -1, 1)
```

Reply:
(I didn't realize at first I could push to your branch)